.travis.yml

```diff
@@ -10,18 +10,26 @@ cache:
 jobs:
   include:
     - stage: Lint and Check types
-      name: Check types
-      script: yarn run check-types & yarn run lint
-
-    - stage: Tests
-      name: Unit
-      script: yarn run test:unit && yarn run codecov
-    -
-      name: e2e
-      script: yarn run test:e2e
-    -
-      name: spec
-      script: yarn run test:spec
+    - name: Lodestar
+      script: lerna run --scope lodestar check-types & lerna run --scope lodestar lint
+    - name: Eth2.0 Types
+      script: lerna run --scope eth2-types check-types & lerna run --scope eth2-types lint
```

Comment:
it will be faster if you run all lint and check-types on all packages in single command. Stage take longer to boot than executing those checks. Same goes for unit tests. e2e and spec should be run each package in it's own stage.
I would maybe suggest to run e2e and spec tests only on master/deployment

Reply:
So here's my thoughts.... I would much rather know the specific package that fails, rather than an outright failure on like all the tests... just my two cents happy to chat it over

Reply:
Because as this grows into like4-5 packages, i would be much happier knowing which one failed. Also we can change this to trigger only if changes were found in specific packages later on.

Reply:
I wonder if we can do serialized tests (parallelization is irrelevant on this) on same machine, because with 4-5 packages we would add like 8-10 mins on CI just for booting machines to run tests